app/views/layouts/application.html.haml

```diff
@@ -41,16 +41,17 @@
   .container
     .row
       = yield
-  :javascript
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-    ga('create', 'UA-100060757-1', 'auto');
-    ga('set', 'anonymizeIp', true);
-    ga('set', 'forceSSL', true);
-    if (#{Rails.env.production?}) {
-      ga('send', 'pageview');
-    }
+  - if Rails.application.config.settings.feature_toggles.google_analytics
+    :javascript
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+      ga('create', 'UA-100060757-1', 'auto');
```

Comment:
moet dit ding eigenlijk niet uit een aparte class komen? of is het wel prima zo

Reply:
volgens mij kan het ook filteren binnen GA, dus dit is wel prima denk ik

Reply:
Check. Dan laat ik het zo.